### PR TITLE
Move Image Controller operator to konflux-ci organization

### PR DESCRIPTION
**Pull request by mmorhun**
This PR uses image released from `103df04d173a93937ca20c4c0baa8571242b28e2` commit, however, it uses deployment definition for the next commit `4f18f640e7d9fbfb1fe3e156156e709c7836254f` in order to make kustomize work.

### CODE REVIEW

**Review by rcerven — APPROVED** [review]

File: hack/preview.sh [hunk 1]
```diff
@@ -204,9 +204,9 @@ sed -i.bak "s/rekor-server.enterprise-contract-service.svc/$rekor_server/" $ROOT
...
 
-[ -n "${IMAGE_CONTROLLER_IMAGE_REPO}" ] && yq -i e "(.images.[] | select(.name==\"quay.io/redhat-appstudio/image-controller\")) |=.newName=\"${IMAGE_CONTROLLER_IMAGE_REPO}\"" $ROOT/components/image-controller/development/kustomization.yaml
-[ -n "${IMAGE_CONTROLLER_IMAGE_TAG}" ] && yq -i e "(.images.[] | select(.name==\"quay.io/redhat-appstudio/image-controller\")) |=.newTag=\"${IMAGE_CONTROLLER_IMAGE_TAG}\"" $ROOT/components/image-controller/development/kustomization.yaml
-[[ -n "${IMAGE_CONTROLLER_PR_OWNER}" && "${IMAGE_CONTROLLER_PR_SHA}" ]] && yq -i e "(.resources[] | select(. ==\"*github.com/redhat-appstudio/image-controller*\")) |= \"https://github.com/${IMAGE_CONTROLLER_PR_OWNER}/image-controller/config/default?ref=${IMAGE_CONTROLLER_PR_SHA}\"" $ROOT/components/image-controller/development/kustomization.yaml
+[ -n "${IMAGE_CONTROLLER_IMAGE_REPO}" ] && yq -i e "(.images.[] | select(.name==\"quay.io/konflux-ci/image-controller\")) |=.newName=\"${IMAGE_CONTROLLER_IMAGE_REPO}\"" $ROOT/components/image-controller/development/kustomization.yaml
```
Comment (chmeliik) [inline hack/preview.sh:207]: Are we already moving the image to quay.io/konflux-ci as well? Or just the repo?

Comment (mmorhun) [reply hack/preview.sh:207]: Yes, correct. I think we can move images location as well.

**Pull request merged**